src/main/java/dao/UserDao.java

```diff
@@ -0,0 +1,13 @@
+package dao;
+
+import java.util.List;
+import model.Bet;
+import model.User;
+
+public interface UserDao {
+    User login(String userName, String password);
+
+    User register(String userName, String password);
+
+    List<Bet> getBetsByUserName(String userName);
```

Comment:
these methods do not really belong to UserDao

dao layer should ONLY have simple methods that directly manipulate storage

`login` and `register` methods should be placed in the class called `AuthenticationService` for example

`getBetsByUserName` i would refactor to just `getUserByUserName`
and when you have the User, you can do whatever you want

I would recommend to remove `login` and `register` now and add just `addUser` method that will simply add User to storage

Reply:
done